src/libraries/System.Net.Quic/src/System/Net/Quic/Internal/ThrowHelper.cs

```diff
@@ -28,27 +30,14 @@ internal static QuicException GetOperationAbortedException(string? message = nul
         return new QuicException(QuicError.OperationAborted, null, message ?? SR.net_quic_operationaborted);
     }
 
-    internal static bool TryGetStreamExceptionForMsQuicStatus(int status, [NotNullWhen(true)] out Exception? exception, bool streamWasSuccessfullyStarted = true, string? message = null)
+    internal static bool TryGetStreamExceptionForMsQuicStatus(int status, [NotNullWhen(true)] out Exception? exception)
```

Comment:
We don't need the special exception here. MsQuic will thrown either ABORTED or INVALID_STATE in case the connection is closed (remotely / locally). It also doesn't use these statuses for anything else.